packages/fondue/src/components/Divider/Divider.tsx

```diff
@@ -32,6 +32,9 @@ const styleMap = {
 
 const DIVIDER_TEST_ID = 'fondue-divider';
 
+/**
+ * @deprecated Please use updated Divider component from @frontify/fondue/components
```

Comment:
Same for `Button` please 😬 

```suggestion
 * @deprecated Please use updated Divider component from `@frontify/fondue/components`.
```